influxdb_client/client/flux_table.py

```diff
@@ -32,6 +32,10 @@ def __str__(self):
         cls_name = type(self).__name__
         return cls_name + "() columns: " + str(len(self.columns)) + ", records: " + str(len(self.records))
 
+    def __repr__(self):
+        """Format for inspection."""
+        return f"<{type(self).__name__}: {len(self.columns)} columns, {len(self.records)} records>"
```

Comment:
Please, use `__repr__` also for `columns` and `records`. Something like:

```suggestion
        return f"<{type(self).__name__}: columns={self.columns.__repr__()}, records={self.records.__repr__()}>"
```